references/synonyms.md

```diff
@@ -1,192 +1,60 @@
 # Synonyms
 
-## List synonyms of one sequence
+The synonyms list is part of the [settings][1] category.
 
-<RouteHighlighter method="GET" route="/indexes/:uid/synonyms/:synonym"/>
+[1]: /references/settings.md
 
-List one sequence and its synonyms in an index.
-
-
-#### Path Variables
-
-| Variable          | Description           |
-|-------------------|-----------------------|
-| **uid**         | The index UID |
-| **synonym**         | Sequence of which the synonyms will be returned |
-
-
-#### Example
-```bash
- curl \
-  -X GET 'http://localhost:7700/indexes/12345678/synonyms/magician'
-```
-
-#### Response: `200 OK`
-
-
-```json
-["harry","merlin"]
-```
-Array of synonyms of the given sequence in the path variable.
-
-## List all sequences and synonyms
+::: tip
+The synonyms list is considered as one resource and has the REST routes in line with this logic.
+:::
 
-<RouteHighlighter method="GET" route="/indexes/:uid/synonyms"/>
+## Get synonyms list
 
-List all sequences and their synonyms in an index.
+<RouteHighlighter method="GET" route="/indexes/:uid/settings/synonyms"/>
 
+Get the list of [synonyms][1]
 
 #### Path Variables
 
 | Variable          | Description           |
 |-------------------|-----------------------|
 | **uid**         | The index UID |
 
-
 #### Example
 ```bash
  curl \
-  -X GET 'http://localhost:7700/indexes/12345678/synonyms'
+  -X GET 'http://localhost:7700/indexes/12345678/settings/synonyms'
 ```
 
 #### Response: `200 OK`
 
 ```json
 {
   "potter": [
-    "harry"
+    "harry",
```

Comment:
Sorry to haven't noticed it before, but I find this example quite weird. For me "harry" is not a synonym for "potter".
```
"hp": [
     "harry potter"
],
"harry potter": [
    "hp"
],
"magician": [
    "harry potter",
    "merlin"
]
``` 
is maybe better?
Plus, we should stay consistent and put everything in lowercase in our examples.